code/style.css

```diff
@@ -1,7 +1,339 @@
+  border: 1px solid gray;
+  background-color:antiquewhite;
+  font-size: 17px;
+  width: 130px;
+  border-radius: 20px;
+  padding: 10px 15px 10px 15px;
+  margin-top: 10px;
+}
+
+button:hover {
+  background-color: #DECDB4;
+}
+
+/* -------------ACCORDION------------------ */
+
+.accordion {
+  border: 2px solid #DECDB4;
+  border-radius: 10px;
+  background-color: #DECDB4;
+  width: 100%;
+  height: auto;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 30px;
+}
+
+.answer {
+  border: 1px solid #F3E9E0;
+  width: 85%;
+  margin: 10px;
+  padding: 0px 20px 0px 20px;
+  text-align: justify;
+  display: none;
+}
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+  
+.question {
+  border-radius: 5px;
+  background-color: #F3E9E0; 
+  width: 90%;
+  padding: 10px 0px 10px 10px;
+  font-size: 20px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+/* This image below is only displayed in deskptop version that is why it displays none in this mobile first design */
+
+.small-image {
+  display: none;
+}
+
+/* --------------footer--------------- */
+
+footer {
+  background-color: #DECDB4;
+    position: relative;
+    font-size: 30px;
+    width: 100%;
+    padding-bottom: 20px;
+    height: 50px;
+    color: white; 
+    text-align: center;
+    object-fit: cover;
+    overflow: hidden;
+}
+
+
+.fa:hover {
+  color: #F3E9E0;
+  font-size: 40px;
+  text-decoration: none;
+}
+
+
+
+
+/* ------------TABLET MEDIA QUERIES-------------- */
+
+
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+
+   .caption {
+       font-size: 25px;
+   }
+
+   .flex-container {
+      max-width: auto;
+      display: flex;
+      flex-direction: column;
+      column-gap: 4em;
+      object-fit: cover;
+      
+  }
+
+  .sign-up-form {
+    width: 100%; 
+    height: 500px;
+  }
+
+  .accordion {
+    width: 100%;
+    height: auto;
+    padding-bottom: 30px;
+  }
+
+}
+
+/* -------------DESKTOP MEDIA QUERIES-------------- */
+
+
+@media screen and (min-width: 1025px) {
+
+  .flex-container {
+      display: flex; 
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+      column-gap: 2em;
+      flex-wrap: wrap;
+      overflow: hidden;
+      object-fit: cover;
+  }
+
+  .sign-up-form {
+  width: 500px; 
+  height: 500px;
+  padding: 30px 10px 20px 10px;
+  margin: 40px 0px 40px 40px;
+  }
+
+
+  .small-image {
+    border: 1px solid #DECDB4;
+    display: block;
+    width: auto;
+    height: 550px;
+    border-radius: 10px;
+  }
+
+
+
+  .accordion {
+    width: 78%;
+    height: auto;
+    padding-bottom: 30px;
+  }
+
+  .question {
+    border-radius: 5px;
+    width: 90%;
+    padding: 10px 10px 10px 10px;
+    font-size: 20px;
+    margin-bottom: 10px;
+  }
+
+  .answer {
+  border: 1px solid #F3E9E0;
+  width: 90%;
+  margin: 10px;
+  }
+
+
+
+  footer {
+    margin-top: 40px;
+    padding-top: 20px;
+  }
```

Comment:
Overall a very nice website, you really have an eye for colors and for layout of content. I have a lot to learn from you. What i would suggest you to do is to use the BEM methodology to name your content. It helps the person reading your code out a lot. And then I would suggest useing the same space everywhere and maybe just have 1 space between each section instead of having multiple and often different spaces =D It gives the reader of the code a cleaner impression. I would also encourage you to go through the code and see if everything you put in it actually fills a purpose (it might do).. But i realised when looking at my own project now that it is so easy to miss stuff that doesnt really have a purpose in the code. Im not saying it is the case for you since i dont know if my inspector tool is working as it should but it doesnt hurt to go through it again.. Love the work you have done a lot! You should be really proud of your ability to create such good content after 2 weeks <3